guides/v9-hammerjs-migration.md

```diff
@@ -91,25 +91,25 @@ from the module. This is automatically done by the migration.
 
 There are other patterns where the deprecated `GestureConfig` is extended, injected or used
 in combination with a different custom gesture config. These patterns cannot be handled
-automatically, but the migration will report such patterns and ask for manual cleanup.
+automatically, but the migration will report such patterns and ask you to perform manual cleanup.
 
 <a name="test-migration"></a>
 ### How to migrate my tests?
 
 Components in your project might use Angular Material components which previously depended
-on HammerJS. There might be unit tests for these components which also test gesture functionality
-of the Angular Material components. For such unit tests, find all failing gesture tests. These
-might need to be reworked to dispatch proper events to simulate gestures, or need to be deleted.
-Specifically gesture tests for the `<mat-slide-toggle>` should be removed. This is because the
-`<mat-slide-toggle>` no longer supports gestures.
+upon HammerJS. There might be unit tests for these components which also test gesture functionality
+of the Angular Material components. For such unit tests, you can find all failing gesture tests.
```

Comment:
I might be wrong here, but previously it was an instruction that asks users to look for failing unit tests. Now, with the new changes, it sounds like a basic statement that people _can_ find the failing ones. Not that they should. Can we make that somehow more clear?

Reply:
Before it seemed to be more of a command, than an instruction. I had changed it to "you should find all failing gesture tests" but then I didn't want to use "should", so I changed it to "can".

However, I've changed it back. It just seems a bit confusing. 

How do people "find all failing gesture tests"? Should it be "identify all failing gesture tests" instead?

Reply:
`Identify all failing gesture tests` definitely works too. It's just that this section should seem like a checklist of things that need to be performed sequentially (if applicable).